### ISSUE TITLE
cron expressions in non-UTC timezones are unsupported

### ISSUE THREAD

**Issue by InsanesTheName** [pitchblack408/pyawscron#62]
Hi! If I'm understanding the below snippet correctly, this enforces that any cron expression passed to `get_all_schedule_bw_dates` must be in UTC. I have a use case where I'd like my cron expressions to be in an arbitrary timezone.

https://github.com/pitchblack408/pyawscron/blob/efa9cad7b8e988816de23e98cd4ae885fa546419/src/pyawscron/awscron.py#L191-L194

Is this something that could be implemented? Or is there a technical limitation which is the reason for that restriction?

Ideally, I'd like to be able to provide a `from_date` and `to_date` which are non-UTC, timezone-aware datetime objects, and a cron expression which specifies occurrences in the same timezone.

**Comment by istrupin** [comment]
I agree it does seem like that validation could be removed to support other timezones
